app/scripts/metamask-controller.test.js

```diff
@@ -1243,90 +1243,6 @@ describe('MetaMaskController', () => {
         );
       });
 
-      it('returns approval from the PermissionsController for only eth_accounts when only permittedChains is specified in params and origin is snapId', async () => {
```

Comment:
should we preserve these tests but redo them as 
`returns approval from the PermissionsController for both eth_accounts and permittedChains when only permittedChains is specified in params and origin is snapId` ?

Reply:
So covering the snap case with the new functionality?